company/templates/company-private-profile-detail.html

```
@@ -0,0 +1,197 @@
+				{% if company.is_address_set %}
+					<a href="{% url 'confirm-company-address' %}" target="_self" class="ed-naked-button-error">
+						Verify your company
+					</a>
+				{% else %}
+					<a href="{% url show_wizard_links|yesno:'company-edit,company-edit-address' %}" target="_self">
+						Set your address
+					</a>
+				{% endif %}
+			</div>
+		</div>
+		{% endif %}
+
+
+		<div class="row-fluid">
+			<div class="span12 great-blue-bar ed-has-edit-button">
+				<h1 class="element-within company-name-title">
+					<span class="dont-break-out">Trade profile for {{company.name}}</span>
+				</h1>
+				<a class="ed-edit-button" href="{% url show_wizard_links|yesno:'company-edit,company-edit-key-facts' %}" style="top: 25px; right: 25px; color: white;">Edit</a>
+			</div>
+		</div>
+
+		<div class="row-fluid company-profile-module-container">
+			<div class="span3 company-logo company-profile-module-inner">
+				<div class="logo-container">
+					<div>
+						{% if company.logo %}
+						<div class="ed-has-edit-button ed-company-profile-logo-container">
+							<img src="{{company.logo}}" />
+							<a class="ed-edit-button" href="{% url 'company-edit-logo' %}" style="bottom: 5px; right: 5px; background: white">Edit</a>
+
+						</div>
+						{% endif %}
+							<a href="{% url 'company-edit-logo' %}" target="_self">
+							{% if not company.logo %}
+								Set logo
+							{% endif %}
+							</a>
+					</div>
+				</div>
+			</div>
+
+			<div class="span5 ed-has-edit-button company-profile-module-inner">
+				<h3>Contact</h3>
+				<a class="ed-edit-button" href="{% url show_wizard_links|yesno:'company-edit,company-edit-contact' %}" style="top: 20px; right: 0;">Edit</a>
+				{% if company.website %}
+					<p><a href="{{ company.website}}">{{ company.website }}</a></p>
+				{% endif %}
+				{% if company.contact_details.email_address %}
+					<p><a href="mailto:{{ company.contact_details.email_address }}">{{ company.contact_details.email_address }}</a></p>
```

This bit will need to be carefully merged. Simplest is probably to keep your version intact, but add the subject line from mine (tests should fail if the subject line is missing). You'll also need to remove the contact link label assert from tests. I used what I was told by Alfie and Rob as the link text, but interesting to see that's not what's in designs! Good food for thought for future grooming sessions.